ntbs-service/wwwroot/source/Components/ConditionalSelectWrapper.ts

```diff
@@ -13,7 +13,11 @@ const ConditionalSelectWrapper = Vue.extend({
         },
         getValueFromChild: function () {
             if (this.$refs["inner-validate"] && this.$refs["inner-validate"].$refs["selectField"]) {
-                return this.$refs["inner-validate"].$refs["selectField"].value;
+                const selectField = this.$refs["inner-validate"].$refs["selectField"];
+                if (selectField.tagName === "DIV") {
```

Comment:
Seems oddly specif - should we not just search the `ref`'s tree for `select always?

Reply:
Initially here (and still in some places) the ref was placed directly on the select tag. I assume this is to workaround that nicely.

Reply:
Yes, the select tag ref isn't accessible anymore once autocomplete has done its magic, so I put it on the containing div instead